src/main/java/com/sawyereffect/steps/StartingSteps.java

```diff
@@ -14,22 +9,13 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.io.IOException;
 import java.util.concurrent.TimeUnit;
 
-public class StartingSteps {
+public class StartingSteps extends DriverFactory{
```

Comment:
Since we don't create the driver here anymore, we need to extend to get the driver used in the execution.